package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "private": true,
-    "workspaces": [
-        "packages/*"
-    ]
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ]
```

Comment:
do not reformat that please, it's excluded from prettier because npm manages that file

Reply:
I didn't format it directly, must have had an old prettier config. I'll revert.